docs/index.adoc

```diff
@@ -359,6 +359,100 @@ entirely, they can save a transaction and gas by calling
 `TBTCVault.unmintAndRedeem` which routes into authorizing the Bridge to
 <<redeeming,redeem>> via `bank.approveBalanceAndCall`.
 
+=== Maintainers
+
+`Bridge` has a set of public knowledge functions that need to be called from
+time to time. A public-knowledge function is where anyone has access to the
+information required to assemble the transaction and the transaction does not
+lead to punishment (slashing).
+
+To compensate for calling these transactions, the caller has the gas spent
+reimbursed by a DAO-funded ETH pool in the same transaction. This works only if
```

Comment:
I would suggest to change this to "the caller **could have** the gas spent reimbursed... This **would** work only if...". I know it's explained later why we don't reimburse direct calling of the `Bridge` functions and why we use maintainers, but someone just skimming the text could think that the calling `Bridge` functions directly is reimbursed.

Reply:
Hmm... I am not sure about it. 

> To compensate for calling these transactions, the caller has the gas spent
reimbursed by a DAO-funded ETH pool in the same transaction.

This is true and it's how it works now in `MaintainerProxy`.

> This works only if
callers follow the order and do not try to front-run each other.

This is true and it's how it works now in `MaintainerProxy`. That's the reason we have `onlyMaintainer` modifier on some functions.

Reply:
Well, yeah, when `Bridge` functions are called via `MaintainerProxy` they get reimbursed. But when they're called directly - they're not. And at this stage of the document we haven't mentioned the `MaintainerProxy` yet, so the assumption is that we're talking about calling the `Bridge` functions directly.

But I know my comment was a bit nitpicky :) I'm ok with leaving the text as it is if you don't think my propositions adds clarity.